basics/getting_started.md

```diff
@@ -39,7 +46,7 @@ how to use the Bazel CI system.
 
       For faster iteration times (but larger binaries), use `//src:bazel-dev.exe` instead.
 
-*  This produces a working Bazel binary in `bazel-bin/src/bazel` (or `bazel-bin/src/bazel.exe` on Windows).
+This produces a working Bazel binary in `bazel-bin/src/bazel` (or `bazel-bin/src/bazel.exe` on Windows).
 
 ## Creating an IntelliJ project
 
```

Comment:
Some of the feedback from this section can also be applied below, such as updating the list to an ordered list for how to work with IntelliJ. Let's meet next week to go over more updates. I'll schedule a working session.

Reply:
Done - I made some updates, but we can work on it next week.